src/main/clj/lemur/core.clj

```diff
@@ -938,13 +939,17 @@ calls launch              - take action (upload files, start cluster, etc)
      true ;default if body is empty
      ~@body))
 
+(defn aws-credentials
+  []
+  (DefaultAWSCredentialsProviderChain.))
+
 (defn -main
   "Run lemur help."
   [& [command & args]]
   (add-command-spec* context init-command-spec)
   (let [[profiles remaining] (split-with #(.startsWith % ":") args)
-        aws-creds (awscommon/aws-credential-discovery)
-        jobdef-path (first remaining)]
+        jobdef-path (first remaining)
+        aws-creds (awscommon/aws-credentials)]
```

Comment:
you add an aws-credentials fn in this namespace at line 942, so I don't think you need to use awscommon/aws-credentials